message_definitions/v1.0/ardupilotmega.xml

```diff
@@ -21,8 +21,8 @@
     <enum name="MAV_CMD">
       <entry value="211" name="MAV_CMD_DO_GRIPPER">
         <description>Mission command to operate EPM gripper.</description>
-        <param index="1">Gripper number (a number from 1 to max number of grippers on the vehicle).</param>
-        <param index="2">Gripper action (0=release, 1=grab. See GRIPPER_ACTIONS enum).</param>
+        <param index="1" label="number" minValue="1" increment="1">Gripper number (a number from 1 to max number of grippers on the vehicle).</param>
```

Comment:
This is consistent, but for some reason a label of "Number" seems a little ambiguous. I'd somewhat prefer "ID", "ID #" or "Gripper ID".
We may need to update labels if there is a preferred capitalisation scheme

Reply:
right now it is consistent with other labels in common.xml

Reply:
OK. Let's see if we get a comment from GCS people, and modify if needed.